opent2t.p.hub/com.wink.hub/js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "opent2t-translator-com-wink-hub",
+  "version": "1.0.5",
+  "description": "Wink Hub",
+  "license": "MIT",
+  "main": "thingTranslator.js",
+  "files": [
+    "thingTranslator.js",
+    "package.json",
+	  "manifest.xml",
+	  "common.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "githttps://github.com/openT2T/translators.git"
+  },
+  "bugs": {
+    "url": "https://github.com/openT2T/translators/issues"
+  },
+  "homepage": "https://github.com/openT2T/translators#readme",
+  "dependencies": {
+    "request": "^2.74.0",
+    "request-promise": "^4.1.1",
+    "opent2t": "^1.0.1",
+    "opent2t-onboarding-org-opent2t-onboarding-winkhub": "^0.0.1",
+    "opent2t-translator-com-wink-lightbulb": "*",
```

Comment:
add min version numbers, use the current translator version and +.0.0.1 it so the current one doesn't get used by accident